src/main/target/STM32_UNIFIED/target.h

```diff
@@ -159,10 +159,6 @@
 #define USE_ACC
 #define USE_GYRO
 
-#define USE_ACC_MPU6050
-#define USE_GYRO_MPU6050
-#define USE_ACC_MPU6500
```

Comment:
Not sure why we would want to remove the MPU6500 driver - this still seems to be a viable part.